CUSTOM.md

```diff
@@ -0,0 +1,218 @@
+  - Set the state to match the element
+  - Stack elements on top of each other as needed
+  - Once all thresholds are set press the "Update Threshold" button (the top one)
+  - Note for thresholds with OR, i.e. Trickster, you'll need to manually edit the json file to add a second threshold at the exact same coordinates as the other one
+- Extra setup:
+  - Create object and place on top of spirit panel
+  - Tag object with "Setup"
+  - doSetup(params)
+    - params: table - contains data about player
+      - color: string - is the color of the player who picked the spirit
+
+Power Cards:
+- Set card onto power card editor
+- Change values that appear to match card
+- Set name of object to that of power card
+- Thresholds:
+  - Grab a threshold token (X) and place it onto object where you want the threshold decal to appear
+  - Then grab elements from the element bag right next to threshold
+  - Set the state to match the element
+  - Stack elements on top of each other as needed
+  - Once all thresholds are set press the "Update Threshold" button (the bottom one)
+  - Note that the decal will appear smaller than the actual size of the object, just a TTS limitation for how small an object can be
+  - Note for thresholds with OR, i.e. Trickster, you'll need to manually edit the json file to add a second threshold at the exact same coordinates as the other one
+
+Aspects:
+- Add "Aspect" tag
+- Set name of object to that of aspect
+- Thresholds:
+  - Place on power card editor
+  - Follow power card threshold instructions
+- Add "Aspect" tag to deck of aspects as well if you have one of those
+
+Blight Cards:
+- Set card onto blight card editor
+- Change values that appear to match card
+- Set name of object to that of blight card
+- If blight card has recurring start of invader effect set object's description to that
+- You'll also want to make sure that Hide Face Down toggle is set for the card
+
+Expansions:
+- Create a bag
+- Name the bag the name of the expansion
+- Add the tag "Expansion" to the bag
+- Decks of cards need to be named one of the following:
+  - "Fear"
+  - "Events"
+  - "Blight Cards"
+  - "Minor Powers"
+  - "Major Powers"
+- Note using single cards in place of a deck may cause setup issues
+
+Adversary & Scenario:
+- Set name of object to that of adversary/scenario
+- collision
+  - workaround for TTS bug https://tabletopsimulator.nolt.io/770
+  - onObjectCollision: boolean - set to true if you have custom object collision code
+  - onObjectCollisionEnter(params)
+    - params: table - contains parameters for callback
+      - hit_object: object - object registered to receive collision events
+      - collision_info: table - table containing data about collision
+  - onObjectCollisionExit(params)
+    - params: table - contains parameters for callback
+      - hit_object: object - object registered to receive collision events
+      - collision_info: table - table containing data about collision
+- broadcast
```

Comment:
Starting message broadcasted as the game starts, can be different for each level.

Reply:
this is a change on current beta version, instead of using a table, there's now a function. the function should return the exact string to be broadcasted